package/MDAnalysis/core/universe.py

```diff
@@ -1289,6 +1289,41 @@ def _fragdict(self):
                 fragdict[a.ix] = fraginfo(i, f)
 
         return fragdict
+    
+    @classmethod
+    def from_smiles(cls, smiles, sanitize=True, addHs=True, **kwargs):
+        """Create a Universe from a SMILES string with rdkit
+
```

Comment:
This is very cool.  Add an example into the docstring.  Also what happens if I pass RDKit a bad SMILES string?

Reply:
RDKit is gonna complain in a not so graceful manner about mol being `None` if we reach `Chem.AddHs(mol)`
I'll add a check on that.

Also, should I use rdkit's camelcase names for the `from_smiles` parameters ? i.e. `AddHs` or `add_hydrogens` ?

Reply:
Might be best to follow RDKit's lead in argument names